install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.11-kube-1.25-api-removals-in-4.12: |
```

Comment:
@bergerhoffer, in [OSDOCS-3571][1], you'd asked to be tagged in this pull request.

[1]: https://issues.redhat.com/browse/OSDOCS-3571

Reply:
Excellent thank you @wking! I'll make a note of this PR for when we update the KBase and 4.12 product docs w/ the actual ack.

Reply:
@wking Do you know if this is live in the latest 4.11 z-stream yet? Just wondering if we can update the kbase articles to include this info yet.